plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -304,6 +304,23 @@ public static void main(String[] args)
         }
     }
 
+    public static class DeltaLakeGlueQueryRunnerMain
+    {
+        public static void main(String[] args)
+                throws Exception
+        {
+            // Set AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables
+            DistributedQueryRunner queryRunner = createDeltaLakeQueryRunner(
+                    DELTA_CATALOG,
+                    ImmutableMap.of("http-server.http.port", "8080"),
+                    ImmutableMap.of("hive.metastore", "glue"));
```

Comment:
Glue comes hand in hand with `s3` . Should we maybe allow `delta.enable-non-concurrent-writes` ?

Reply:
I didn't put it because we can set the property in VM options in IntelliJ.